django/db/models/fields/__init__.py

```diff
@@ -1085,6 +1085,8 @@ def formfield(self, **kwargs):
         # will be validated twice. This is considered acceptable since we want
         # the value in the form field (to pass into widget for example).
         defaults = {'max_length': self.max_length}
+        if self.null and not connection.features.interprets_empty_strings_as_nulls:
```

Comment:
What is the correct way to lookup the connection in this situation? I based this code on other feature checks in the same file. For example:

* <https://github.com/django/django/blob/a8f957797d8035a542cdb20b03aaebd81b9529e2/django/db/models/fields/__init__.py#L291>
* <https://github.com/django/django/blob/a8f957797d8035a542cdb20b03aaebd81b9529e2/django/db/models/fields/__init__.py#L771>

Are these examples handling multi-db support correctly? I'm not sure exactly how to proceed here. Any advice is welcome, thanks.